ws-client/Cargo.toml

```diff
@@ -8,12 +8,13 @@ license = "MIT"
 
 [dependencies]
 async-trait = "0.1"
-async-std = { version = "1.9", default-features = false, features = ["std"] }
+async-std = "1.9"
```

Comment:
sorry I made I mistake here, didn't realize that when default-features are disabled, the `net` module is not enabled and can't be enabled by any feature flag https://github.com/async-rs/async-std/blob/master/src/lib.rs#L314-#L318

Reply:
the feature is probably leaked from somewhere else and works in the workspace